### PR TITLE
Add Navbar to Storybook 

### LINKED ISSUE #1955: [FEATURE] Add Navbar to Storybook
### Description

components/navbar/Navbar.js in #1944

I want to give this a try, please assign me to it :)

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by zhaobenny**
## Fixes Issue
Closes #1955 

## Changes proposed
Addition of components/navbar/navbar.stories.js

Adds Navbar to StoryBook.

Made Storybook look in ../public for static assets

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
image

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1960"></a>

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = {
     "@storybook/addon-interactions",
     "storybook-addon-next",
   ],
+  staticDirs: ['../public'],
```
Comment (eddiejaoude) [inline .storybook/main.js:12]: Not sure this is needed?

Comment (zhaobenny) [reply .storybook/main.js:12]: Without this line, the picture in the corner is broken 

image

Comment (eddiejaoude) [reply .storybook/main.js:12]: oh I see, thank you for clarifying 👍

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**